src/PIL/GifImagePlugin.py

```diff
@@ -28,12 +28,22 @@
 import math
 import os
 import subprocess
+from enum import IntEnum
 
 from . import Image, ImageChops, ImageFile, ImagePalette, ImageSequence
 from ._binary import i16le as i16
 from ._binary import o8
 from ._binary import o16le as o16
 
+
+class ModeStrategy(IntEnum):
+    AFTER_FIRST = 0
+    ALWAYS = 1
+    DIFFERENT_PALETTE_ONLY = 2
+
+
+PALETTE_TO_RGB = ModeStrategy.AFTER_FIRST
```

Comment:
Can this too, if it's part of the public API? Some suggestions here, but let's worry about it too much. https://stackoverflow.com/q/20227051/724176

Reply:
Ok, I picked one from that link. See what you think.

Reply:
Hmm, the `#: versionadded:: 9.1.0` doesn't get expanded:

<img width="557" alt="image" src="https://user-images.githubusercontent.com/1324225/160376567-dc45351d-1d9b-4d0f-8078-2217dcd946f9.png">

https://pillow--6150.org.readthedocs.build/en/6150/reference/plugins.html#module-PIL.GifImagePlugin

Reply:
Ok, it should be fixed now.